roles/kubernetes/node/tasks/install.yml

```diff
@@ -67,9 +63,3 @@
   tags:
     - hyperkube
     - upgrade
-
-- name: install | Copy socat wrapper for Container Linux
```

Comment:
again why removing socat?

Reply:
Because Socat is already part of container linux so no need to download it